.travis.yml

```diff
@@ -1,14 +1,37 @@
+---
+os: osx
 sudo: true
-git:
-  depth: 10
+# lie to make travis work with nvm on osx
+language: cpp
+env:
+  matrix:
+    - TRAVIS_NODE_VERSION="0.10"
+    - TRAVIS_NODE_VERSION="0.12"
+    - TRAVIS_NODE_VERSION="2"
+    - TRAVIS_NODE_VERSION="4"
+    - TRAVIS_NODE_VERSION="5"
```

Comment:
This confuses me even further then as I did this to make sure the consist env was working as it was always erroring out before on osx not matter what was done as discussed previously.. It's the fact that I'm installing a global version of sass-lint and testing that the npm get prefix command is working.. Without specifying these node versions it fails every time on Travis.